src/main/java/Car.java

```diff
@@ -0,0 +1,29 @@
+import java.io.StreamCorruptedException;
+
+public class Car {
+    private String name;
+    private int score;
+    private RandomNumberGenerator randomNumberGenerator = new RandomNumberGenerator();
+
+    public Car(String name){
+        this.name = name;
+    }
+
+    public Car(String name, RandomNumberGenerator randomNumber){
+        this.name = name;
+        this.randomNumberGenerator = randomNumber;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public int getScore(){
+        return score;
+    }
+
+    public void move(){
+        int randomNumber = randomNumberGenerator.generate();
+        if (randomNumber > 4)   score++;
+    }
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used
컨벤션도 적용해주세요!
```suggestion
    public void move() {
        int randomNumber = randomNumberGenerator.generate();
        if (randomNumber > 4) {
           score++;
       }
    }
```